SPECS/elfutils/elfutils.spec

```diff
@@ -210,14 +217,15 @@ fi
 %doc README TODO CONTRIBUTING
 %{_bindir}/eu-*
 %{_bindir}/debuginfod-find
-%exclude %{_bindir}/eu-ld
+#%exclude %{_bindir}/eu-ld
```

Comment:
Why comment it out instead of removing the line? Ideally, let's also remove the two other commented lines below.

Reply:
Makes sense